Monika After Story/game/cardgames.rpy

```diff
@@ -0,0 +1,4781 @@
+            m "Ahaha! My luck is back~"
+
+        else:
+            m "There we go!"
+            m "I finally won you~"
+
+    elif store.mas_nou.player_win_streak > 2:
+        $ choice = renpy.random.randint(0, 2)
+
+        if not choice:
+            m "Don't relax, [player]~"
+
+        elif choice == 1:
+            m "I won!"
+
+        else:
+            m "Yay I won this time!~"
+
+    elif store.mas_nou.monika_win_streak > 5:
+        $ choice = renpy.random.randint(0, 2)
+
+        if not choice:
+            m "And another win for me!~"
+
+        elif choice == 1:
+            if len(store.mas_nou.game.player.hand) < 3:
+                m "That was tough, [player]!"
+                extend "You almost win this time."
+
+            else:
+                m "I have a feeling that you'll win next [_round!t]."
+
+        else:
+            if len(store.mas_nou.game.player.hand) < 3:
+                m "Well played, [player]. But the win is mine again~"
+
+            else:
+                m "That was fun!"
+                m "I hope you're enjoying playing with me, [player]~"
+                m "Maybe next time you'll win."
+
+    elif store.mas_nou.monika_win_streak > 2:
+        $ choice = renpy.random.randint(0, 2)
+
+        if not choice:
+            m "Ehehe~ Another win for me~"
+
+        elif choice == 1:
+            if len(store.mas_nou.game.player.hand) < 3:
+                m "You were quite close this time, [player]."
+
+            else:
+                m "Should I go easy on you?"
+                m "Ahaha, just kidding, [player]~"
+
+        else:
+            m "I won again!"
+
+    elif store.mas_nou.game.total_turns < 25:
+        if store.mas_nou.monika_win_streak > 0:
+            $ choice = renpy.random.randint(0, 2)
+
+            if not choice:
+                m "Another quick win for me!"
+
+            elif choice == 1:
+                m "Can't keep up with me, huh?~"
+
+            else:
+                m "Yay, I won again!"
+
+        else:
+            $ choice = renpy.random.randint(0, 1)
+
+            if not choice:
+                m "Yay, I won~"
+
+            else:
+                m "That was quick."
+
+    elif store.mas_nou.game.total_turns > 75:
+        $ choice = renpy.random.randint(0, 2)
+
+        if not choice:
+            m "That was a long [_round!t]!"
+
+            if len(store.mas_nou.game.player.hand) < 4:
+                if store.mas_nou.monika_win_streak > 0:
+                    m "You even almost win this time."
+
+                else:
+                    m "You even almost win."
+
+                m "Ehehe~ Well played!"
+
+            else:
+                m "Well played!"
+
+        elif choice == 1:
+            m "That was intense!"
+
+        else:
+            if len(store.mas_nou.game.player.hand) > 4:
+                m "Not bad, [player]."
+                m "I think you even could win this time if not all those card you've drawn."
```

Comment:
Grammar is a bit off in places, you probably meant "I think you could even have won that time, if not for all those cards you drew."

Reply:
Are you sure about `that time`? Maybe `this time`?